mne/bem.py

```diff
@@ -2187,3 +2187,62 @@ def check_seghead(surf_path=op.join(subj_path, 'surf')):
             incomplete=incomplete, extra=msg)
         write_bem_surfaces(dec_fname, dec_surf, overwrite=overwrite)
     logger.info('[done]')
+
+
+@verbose
+def distance_to_bem(pos, bem, trans=None, verbose=None):
+    """Calculate the distance of positions to inner skull surface.
+
+    Parameters
+    ----------
+    pos : array, shape (3,) | (n, 3)
+        Position(s) in m, in head coordinates.
+    bem : instance of ConductorModel
+        Conductor model.
+    %(trans)s If None (default), assumes bem is in head coordinates.
+    %(verbose)s
+
+    Returns
+    -------
+    distances : float | array, shape (n,)
+        The computed distance(s). A float is returned if pos is
+        an array of shape (3,) corresponding to a single position.
+    """
+    ndim = pos.ndim
+    if ndim == 1:
+        pos = pos[np.newaxis, :]
+
+    n = pos.shape[0]
+    distance = np.zeros((n,))
+
+    logger.info(
+        f'Computing distance to inner skull surface for {n} positions...'
```

Comment:
Use `mne.utils._pl` here like
```
{n} position{_pl(n)}...
```
It puts a `'s'` if it's != 1 and `''` if it `== 1`.